backend/app/debug.js

```diff
@@ -1,7 +1,27 @@
 import debug from 'debug';
 
+let counters = {};
```

Comment:
What is the purpose of this counter?

Reply:
It's used by `debug.count()` (and `countReset`).

The idea is to mimic the `console` object. It's something I started privately and copy/pasted to this project. `count`/`countReset` is not currently used in this project.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Console/count